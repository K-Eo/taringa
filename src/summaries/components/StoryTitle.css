.StoryTitle {
  font-size: 90%;
  margin-bottom: 0 !important;
  margin-right: 0.25rem !important;
  margin-top: 0 !important;
  overflow: hidden;
  text-overflow: ellipsis;
  white-space: nowrap;
  word-break: break-word;
}

.item-big .StoryTitle,
.item-medium .StoryTitle {
  font-size: 95%;
  overflow: initial;
  text-overflow: initial;
  white-space: initial !important;
}

.item-big .StoryTitle {
  font-weight: 400;
  line-height: 20px;
  margin-top: 0.25rem !important;
  order: 2;
}
