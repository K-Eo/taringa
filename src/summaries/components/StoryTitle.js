import React from 'react'

import './StoryTitle.css'

const StoryTitle = ({ children }) => {
  return (
    <p className="StoryTitle" title={children}>
      {children}
    </p>
  )
}

export default StoryTitle
