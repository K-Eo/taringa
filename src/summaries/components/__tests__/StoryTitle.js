import React from 'react'

import StoryTitle from '../StoryTitle'

describe('StoryTitle', () => {
  it('exists', () => {
    expect(StoryTitle).toBeDefined()
  })

  it('renders', () => {
    const wrapper = shallow(<StoryTitle>foobar</StoryTitle>)

    expect(wrapper).toMatchSnapshot()
  })
})
