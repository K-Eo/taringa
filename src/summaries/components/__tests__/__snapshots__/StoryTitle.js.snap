// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StoryTitle renders 1`] = `
<p
  className="StoryTitle"
  title="foobar"
>
  foobar
</p>
`;
